doc/source/methods/tabulardrift.ipynb

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
<p>Rouge <code>cat</code>  in the constructor example!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>